handlers/other/stash_remover.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby1.9.3
+#
+# Sensu Handler: stash_remover
+#
+# This handler removes stashes of services coming back to OK.
```

Comment:
Can you expand this header based on the style guide here: https://github.com/sensu/sensu-community-plugins/blob/master/CONTRIBUTING.md#coding-style